target/linux/mvebu/patches-4.19/544-add-armada-3720-espressobin-ultra.dts.patch

```diff
@@ -0,0 +1,9 @@
+--- a/arch/arm64/boot/dts/marvell/Makefile
++++ b/arch/arm64/boot/dts/marvell/Makefile
+@@ -1,5 +1,6 @@
+ # SPDX-License-Identifier: GPL-2.0
+ # Mvebu SoC Family
+ dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-db.dtb
+ dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-espressobin.dtb
++dtb-$(CONFIG_ARCH_MVEBU) += armada-3720-espressobin-ultra.dtb
+ dtb-$(CONFIG_ARCH_MVEBU) += armada-7040-db.dtb
```

Comment:
Remove this patch and following one, these are not necessary.

Reply:
Removed both patches.